setup.cfg

```diff
@@ -2,4 +2,16 @@
 universal=1
 
 [flake8]
+exclude=*migrations*
```

Comment:
Why should migrations be excluded? (we didn't see a need to exclude them in Django)